docs/additional_reading/heroku_deployment.md

```diff
@@ -3,7 +3,9 @@ The generator has created the necessary files and gems for deployment to Heroku.
 
 + `Procfile`: used by Heroku and Foreman to start the server
 + `.buildpacks`: used to install Ruby and Node environments
++ `config/unicorn.rb`: Unicorn config file
```

Comment:
@justin808 I think @robwise has been reading to many article about dying unicorns posted in our slack channel and he really meant `config/puma.rb` :)